# HG changeset patch
# User Wolfgang Rosenauer <wr@rosenauer.org>
# Parent ba2d9b1374e15e7c339a44452169d0afac2ca82e
With openSUSE 11.4 the desktop file name changed from MozillaFirefox to firefox

diff --git a/browser/components/preferences/advanced.js b/browser/components/preferences/advanced.js
--- a/browser/components/preferences/advanced.js
+++ b/browser/components/preferences/advanced.js
@@ -751,17 +751,17 @@ var gAdvancedPane = {
     if (kde_session == 1) {
       var shellObj = Components.classes["@mozilla.org/file/local;1"]
                                .createInstance(Components.interfaces.nsILocalFile);
       shellObj.initWithPath("/usr/bin/kwriteconfig");
       var process = Components.classes["@mozilla.org/process/util;1"]
                               .createInstance(Components.interfaces.nsIProcess);
       process.init(shellObj);
       var args = ["--file", "kdeglobals", "--group", "General", "--key",
-          "BrowserApplication", "MozillaFirefox"];
+          "BrowserApplication", "firefox"];
       process.run(false, args, args.length);
     }
     let selectedIndex =
       shellSvc.isDefaultBrowser(false, true) ? 1 : 0;
     document.getElementById("setDefaultPane").selectedIndex = selectedIndex;
   }
 #endif
 };
